HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -310,6 +310,47 @@ code::
 "mississippi".split("i").postcs;
 ::
 
+method::insertAll
+Return a new collection of the same kind, with all elements of another collection or object inserted, starting from a given index.
```

Comment:
if you talk about equality, maybe add the actual operandas well? `strong::equality:: / code::==::`